src/helpers/code-builder.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-var util = require('util')
+const util = require('util')
```

Comment:
Would be great to enforce this with ES Lint (now or eventually)

Reply:
@develohpanda I have a WIP PR that'll do that and can get that work a PRable state after this is accepted.